pkg/security/controller/controller.go

```diff
@@ -65,6 +66,7 @@ func (c *Allocation) Next(ns *kapi.Namespace) error {
 	}
 	tx.Add(func() error { return c.uid.Release(block) })
 	ns.Annotations[security.UIDRangeAnnotation] = block.String()
+	ns.Annotations[security.FSGroupAnnotation] = strconv.FormatUint(uint64(block.Start), 10)
```

Comment:
Yes this is fine.  We will eventually have a different strategy configurable by an admin that changes this up.  When we get user namespaces we might want to allocate only FSGroup.